ExampleNet45Project/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+
+namespace Example
+{
+    internal class Program
```

Comment:
Really like the simplification you did with these examples. Will you add back the other 'kitchen sink' examples from `ExampleNet45Project/Example.cs` to this file? They don't need to be in both the ExampleCoreProject and `ExampleNet45Project`, but they should be in one of them. They make more sense here, than amongst the `DependencyInjection` usage example.

Reply:
@eshanholtz I put back the 'kitchen sink' examples back to `ExampleNet45Project/Example.cs`, thanks for your recommendations.